CommitPoolContract/contracts/SinglePlayerCommit.sol

```diff
@@ -223,146 +136,64 @@ contract SinglePlayerCommit is ChainlinkClient, Ownable {
         uint256 _stake,
         uint256 _depositAmount,
         string memory _userId
-    ) public returns (bool success) {
-        require(deposit(_depositAmount), "SPC::depositAndCommit - deposit failed");
-        require(
-            makeCommitment(_activityKey, _goalValue, _startTime, _endTime, _stake, _userId),
-            "SPC::depositAndCommit - commitment creation failed"
-        );
-
-        return true;
+    ) public noCommitment returns (bool commitmentExists) {
+        require(_deposit(_depositAmount));
+        _makeCommitment(_activityKey, _goalValue, _startTime, _endTime, _stake, _userId);
+        return commitments[msg.sender].exists;
     }
 
-    /// @notice Enables processing of open commitments after endDate that have not been processed by creator
-    /// @param committer address of the creator of the committer to process
-    /// @dev Process commitment by lookup based on address, checking metrics, state and updating balances
-    function processCommitment(address committer) public {
-        console.log("Processing commitment");
-        require(commitments[committer].exists, "SPC::processCommitment - commitment does not exist");
-        Commitment storage commitment = commitments[committer];
-
-        require(commitment.endTime < block.timestamp, "SPC::processCommitment - commitment is still active");
-        require(commitment.endTime < commitment.lastActivityUpdate, "SPC::processCommitment - update activity");
-
-        require(_settleCommitment(commitment), "SPC::processCommitmentUser - settlement failed");
-
-        emit CommitmentEnded(committer, commitment.met, commitment.stake);
+    function processAndSettle(address commitmentOwner) public returns (bool success){
```

Comment:
in various places we refer to the address that has made a committment as either `committer` or `committmentOwner`. We should be consistent in how we name this. I have a slight preference for the former, but don't really care as long as we're consistent throughout.

Reply:
Good point, I'll harmonize them